CMakeLists.txt

```diff
@@ -77,9 +77,9 @@ cmake_policy(POP)
 ######## LLVM rules for installed libraries ###########
 # We want to select the latest stable release even if others are installed
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-  find_package(LLVM 14.0.0 REQUIRED CONFIG)
+  find_package(LLVM 14.0 REQUIRED CONFIG)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-  find_package(LLVM 14.0.0 REQUIRED CONFIG)
+  find_package(LLVM 14.0 REQUIRED CONFIG PATHS /usr/local/opt)
```

Comment:
Do we need to hardcode this path?

Reply:
No.  I think this should be setup in the shell path by bootstrap - which I think it is.   It was just a workaround for a situation where I didn't want to rerun bootstrap and I forgot to revert it before pushing.

Reply:
Okay, would you like me to remove it before merging or would that break things for you?

Reply:
Okay, would you like me to remove it before merging or would that break things for you?